src/hotspot/share/opto/phaseX.hpp

```diff
@@ -616,6 +617,10 @@ class PhaseIterGVN : public PhaseGVN {
     // '-XX:VerifyIterativeGVN=1000'
     return ((VerifyIterativeGVN % 10000) / 1000) == 1;
   }
+  static bool is_verify_extra() {
+    // '-XX:VerifyIterativeGVN=10000'
+    return ((VerifyIterativeGVN % 100000) / 10000) == 1;
+  }
```

Comment:
You will need to add extra documentation to the flag. And also there is a test that uses the flag. You should adjust it to enable this bit as well.